### PR TITLE
Fix import path

### PR DESCRIPTION
**Pull request by thibaultmeyer**
According to #135, this pull request fix all imports. Now the commands `go get all` and `go build` works like a charm.

**Note:** Please, don't hesitate to clone this branch to test in your own computer.

```
$ make build
go build -v -tags '' -ldflags '-s -w -X "github.com/go-gitea/gitea/modules/setting.BuildTime=2016-11-11 08:40:08 UTC" -X "github.com/go-gitea/gitea/modules/setting.BuildGitHash=243e0b5"' -o bin/gitea
github.com/go-gitea/gitea/models
github.com/go-gitea/gitea/modules/template
github.com/go-gitea/gitea/modules/auth
github.com/go-gitea/gitea/modules/cron
github.com/go-gitea/gitea/modules/ssh
github.com/go-gitea/gitea/routers/api/v1/convert
github.com/go-gitea/gitea/modules/context
github.com/go-gitea/gitea/routers/repo
github.com/go-gitea/gitea/routers/api/v1/misc
github.com/go-gitea/gitea/routers/dev
github.com/go-gitea/gitea/routers/user
github.com/go-gitea/gitea/routers/api/v1/repo
github.com/go-gitea/gitea/routers/api/v1/user
github.com/go-gitea/gitea/routers
github.com/go-gitea/gitea/routers/org
github.com/go-gitea/gitea/routers/api/v1/org
github.com/go-gitea/gitea/routers/api/v1/admin
github.com/go-gitea/gitea/routers/admin
github.com/go-gitea/gitea/routers/api/v1
github.com/go-gitea/gitea/cmd
github.com/go-gitea/gitea

$
```

### CODE REVIEW

**Comment by strk** [comment]
Sounds like a revert of #135, basically, no ?

**Comment by thibaultmeyer** [comment]
Yeah, it seems that #135 not working rather than TravisCI return success (see comments from @tboerger on #135)

**Comment by strk** [comment]
See #141, should work once https://github.com/go-gitea/redirects/pull/5 is merged

**Comment by lunny** [comment]
no. I think we should fix the wrong import path, not revert.

File: models/issue.go [hunk 1]
```diff
@@ -167,9 +167,9 @@ func (issue *Issue) HTMLURL() string {
 // State returns string representation of issue status.
 func (i *Issue) State() api.StateType {
 	if i.IsClosed {
-		return api.STATE_CLOSED
```
Comment (stevenroose) [inline models/issue.go:170]: These changes have nothing to do with import paths..

**Pull request closed without merging**